client/web-sveltekit/src/lib/repo/RepoPopover/RepoPopover.svelte

```diff
@@ -0,0 +1,312 @@
+    $: author = repo.commit?.author.person.name
+    $: commitDate = repo.commit?.author.date
+    $: avatar = repo.commit?.author.person
+    $: codeHostKind = repo.externalServices.nodes[0].kind
+    $: codeHostIcon = getCodeHostIcon(codeHostKind)
+</script>
+
+<div class="root">
+    {#if withHeader}
+        <div class="header">
+            <div class="icon-name-access">
+                <!-- @TODO: We need to use our customer's logo here, not the code host's -->
+                <!--Icon svgPath={mdiGitlab} /-->
+                <h4 class="repo-name">{repo.name}</h4>
+                <div class="access">
+                    <small>{repo.isPrivate ? 'Private' : 'Public'}</small>
+                </div>
+            </div>
+            <div class="code-host">
+                <Icon svgPath={codeHostIcon} --color="var(--text-body)" --size={24} />
+                <div><small>{capitalize(codeHostKind)}</small></div>
+            </div>
+        </div>
+        <div class="divider" />
+    {/if}
+    <div class="description-and-tags">
+        <div class="description">{repo.description}</div>
+        <div class="tags">
+            {#if repo.tags.nodes.length > 0}
+                {#each repo.tags.nodes as tag}
+                    <div class="tag"><small>{tag.name}</small></div>
+                {/each}
+            {/if}
+        </div>
+    </div>
+    <div class="divider" />
+    <div class="last-commit">
+        <div class="title-and-commit">
+            <div class="title">
+                <small>Last Commit</small>
+            </div>
+            <div class="commit-and-number">
+                <div class="commit">
+                    <small>{subject}</small>
+                </div>
+                {#if commitNumber}
+                    <div class="number">
+                        <small>{commitNumber}</small>
+                    </div>
+                {/if}
+            </div>
+        </div>
+        <div class="commit-info">
+            <div class="author-and-time">
+                {#if avatar}
+                    <Avatar {avatar} --avatar-size="1.0rem" />
+                {/if}
+                <div class="author">
+                    <small>{author}</small>
+                </div>
+                <div class="separator">{CENTER_DOT}</div>
+
+                {#if commitDate}
+                    <div class="commit-date">
+                        <small>{formatDistanceToNow(commitDate, { addSuffix: false })}</small>
+                    </div>
+                {/if}
+            </div>
+        </div>
+    </div>
+    <div class="divider" />
+    <div class="repo-stats">
+        <div class="stats">
+            <div class="stat"><small>{repo.language}</small></div>
+        </div>
+        <div class="stat">
+            <Icon svgPath={mdiStarOutline} size={16} style="margin-right: 0.15rem;" />
+            <small>{formatNumber(repo.stars)}</small>
+        </div>
+    </div>
```

Comment:
Overall this component seems to be doing a lot possibly unnecessary element nesting which makes it quite "verbose".

This here is one example. I think this could be written as

```suggestion
    <div class="footer">
        <small>{repo.language}</small>
        <div class="star">
            <Icon svgPath={mdiStarOutline} size={16} style="margin-right: 0.15rem;" />
            <small>{formatNumber(repo.stars)}</small>
        </div>
    </div>
```

(and this can be even more reduced if you use the `RepoStar` component).

In general it's good to use as few DOM/HTML elements as possible.

I suggest to go through the template and remove unnecessary wrappers, especially around single elements.